res/values/strings.xml

```diff
@@ -311,9 +311,9 @@
     <string name="welcome_intro1_message">The messenger with the broadest audience in the world. Free and independent.</string>
     <string name="welcome_start_messaging">Start messaging</string>
     <string name="login_title">Log in</string>
-    <string name="login_header">Log in to your server</string>
-    <string name="login_explain">Log in with an existing email account</string>
-    <string name="login_subheader">For known email providers additional settings are set up automatically. Sometimes IMAP needs to be enabled in the web settings. Consult your email provider or friends for help.</string>
+    <string name="login_header">Log in to your email account/string>
+    <string name="login_explain">Supply your email address and the password for it</string>
+    <string name="login_subheader">Let us know what additional settings are required, if not filled in automatically. Sometimes turning on IMAP in the web settings of your provider is needed too.</string>
```

Comment:
why remove the friend part? in general, please not that there are already lots of user-feedback incorporated in the wordings. so, apart from typos or so, i am a bit careful in changing things :) also as this would require all translations to be redone.

Reply:
This is user feedback. It is the most recent user feedback.
This neither is nor steers the user towards any documentation that would help.
"The web settings" of where? If as much was clear, there would be less of a reason to direct the user towards getting help. I don't know of any e-mail providers that will help with that. "Hey I am trying to set up Delta Chat" is possibly not good PR. Asking friends for help is what people do anyway?